gammapy/maps/region/geom.py

```diff
@@ -853,11 +852,7 @@ def plot_region(self, ax=None, kwargs_point=None, path_effect=None, **kwargs):
             kwargs.setdefault("facecolor", "None")
             kwargs.setdefault("edgecolor", "tab:blue")
             kwargs_point.setdefault("marker", "*")
-
-            for key, value in kwargs.items():
-                key_point = ARTIST_TO_LINE_PROPERTIES.get(key, None)
-                if key_point:
-                    kwargs_point[key_point] = value
+            kwargs_point.setdefault("color", "tab:blue")
```

Comment:
```suggestion
            kwargs_point_default = kwargs.copy()            
            kwargs_point_default.update(kwargs_point)

            for key, value in kwargs_point_default.items():
                key_point = ARTIST_TO_LINE_PROPERTIES.get(key, None)
                if key_point and key_point not in kwargs_point:
                    kwargs_point[key_point] = value
                    
            if "color" in kwargs:
                kwargs_point.setdefault("color", kwargs["color"])
            if "color" in kwargs_point:
                kwargs_point["markeredgecolor"] = kwargs_point["color"]
```

Reply:
so in the example :

```
import matplotlib.pyplot as plt
from gammapy.datasets import MapDataset
from gammapy.catalog import SourceCatalog3FHL
fermi_dataset = MapDataset.read("$GAMMAPY_DATA/fermi-3fhl-gc/fermi-3fhl-gc.fits.gz", name="fermi_dataset")
catalog = SourceCatalog3FHL()
models = catalog.to_models().select_from_geom(fermi_dataset.geoms["geom"])
ax = fermi_dataset.excess.sum_over_axes().smooth("0.2 deg").plot(add_cbar=True, cmap="Blues")
ax = models.plot_regions(ax=ax, linewidth=1,  color="red")

```

points should also be red and not blue

Reply:
and 

```
ax = models.plot_regions(ax=ax, linewidth=1,  color="red", edgecolor="green"
kwargs_point={"marker":"d", "markersize":5, "markeredgecolor":"green"})
```
will be red because color has priority over edgecolor as for the lines